test/external/CMakeLists.txt

```diff
@@ -18,7 +18,15 @@ FetchContent_Declare(
     SOURCE_DIR ${PROJECT_SOURCE_DIR}/external/tests/testcorenrn
 )
 
-FetchContent_MakeAvailable(ringtest testcorenrn)
+FetchContent_Declare(
+    nrntest
+    GIT_REPOSITORY https://github.com/neuronsimulator/nrntest
+    GIT_TAG 57fd987
```

Comment:
not to forget: this is a branch

Reply:
I assume here `GIT_TAG` accepts branch name as well?

Reply:
haven't tested, but it should

Reply:
Following doc:
> Git branch name, tag or commit hash.